hawkbit-rest/hawkbit-mgmt-api/src/main/java/org/eclipse/hawkbit/mgmt/rest/api/MgmtRestConstants.java

```diff
@@ -275,7 +275,7 @@ public final class MgmtRestConstants {
      * Request parameter if the artifact url handler should be used
      */
     public static final String REQUEST_PARAMETER_USE_ARTIFACT_URL_HANDLER = "useartifacturlhandler";
-
+    public static final String REQUEST_PARAMETER_WITH_DOWNLOAD_URL = "withdownloadurl";
```

Comment:
why do we need that? isn't it sufficient to append "?representation=full&useartifacturlhandler=true"?

Reply:
Good point! I made of use the representation model and useartifacturlhandler parameter.